test/www/jxcore/test/mockmobilegetdocpath.js

```diff
@@ -0,0 +1,17 @@
+var errArg;
+var fileLocationArg;
+
+function MobileGetDocumentsPath() {
+  return;
+}
+
+MobileGetDocumentsPath.storeGetDocumentsPathReturnArguments = function () {
+  errArg = arguments[0];
```

Comment:
Why are you using arguments? Why not just put errArg and fileLocationArg into the function signature?